actions/deploy-renku/deploy-dev-renku.py

```diff
@@ -156,26 +150,22 @@ def configure_requirements(tempdir, reqs, component_versions):
         help="Namespace for this release",
         default=os.environ.get("RENKU_NAMESPACE"),
     )
-    parser.add_argument(
-        "--release", help="Release name", default=os.environ.get("RENKU_RELEASE")
-    )
+    parser.add_argument("--release", help="Release name", default=os.environ.get("RENKU_RELEASE"))
     parser.add_argument(
         "--anonymous-sessions",
-        help="Enable anonymous sessions by setting to \"true\"",
+        help='Enable anonymous sessions by setting to "true"',
```

Comment:
is this formatted with `black`?

Reply:
Yes, do we adopt something else?

Reply:
no, I was just surprised that the double quotes were replaced with single quotes...

Reply:
ah, yes, it's an exception for when a string contains double quotes within: instead of escaping Black prefers to surround them with single quotes.